concourse/model/traits/image_scan.py

```diff
@@ -299,6 +299,9 @@ def for_cve(self, cve_score: float):
         else:
             return self.low
 
+    def for_cfg_element(self):
+        return self.high
```

Comment:
this is not a good location for that - consider the module name..

just hard-code - as also done, e.g. for clamav-scans - the severities, based on cfg-element-policy-violation. also, we likely will want to use different priorities (no-owner might have different priority than overdue rotation)